src/storage/src/hummock/store/local_hummock_storage.rs

```diff
@@ -321,6 +322,19 @@ impl LocalStateStore for LocalHummockStorage {
         .await
     }
 
+    async fn try_flush(&mut self) -> StorageResult<()> {
+        if self.mem_table.kv_size.size() > MEM_TABLE_SPILL_THRESHOLD {
+            tracing::info!(
+                "The size of mem table exceeds 64 Mb and spill occurs. table_id {}",
+                self.table_id.table_id()
+            );
+
+            self.flush(vec![]).await?;
+        }
+
+        Ok(())
```

Comment:
Shouldn't it return `Err` when it's not flushed, given the method name is called `try_flush`?

Reply:
We may call it `may_flush` because `try_xx` interface always has error handling branch, but here we just want to flush or ignore.